miner/worker.go

```diff
@@ -212,6 +216,28 @@ func (w *worker) commitNewWork() (*types.Block, error) {
 	return w.commit(env)
 }
 
+// enforcePredicates takes a set of pending transactions (grouped by sender, and ordered by nonce) and returns a
+// subset of those transactions (grouped by sender) that satisfy predicateContext.
+// Any transaction sent by a given sender that is received after a transaction that does not
+// satisfy predicateContext is removed from the TxPool and is not included in the return value.
+func (w *worker) enforcePredicates(rules params.Rules, predicateContext *precompile.PredicateContext, pending map[common.Address]types.Transactions) map[common.Address]types.Transactions {
```

Comment:
Nit: (Optional) Not sure what your normal linting rules are, but I think it'd be easier to read if this function signature was split across multiple lines since its 188 characters.